trino/trino_test.go

```diff
@@ -410,6 +411,22 @@ func TestTypeConversion(t *testing.T) {
 			ResponseUnmarshalledSample: nil,
 			ExpectedGoValue:            nil,
 		},
+		{
+			// rows return data as-is for slice scanners
```

Comment:
Is that ok? What is the alternative?

Reply:
This is ok, it's similar to how arrays are handled, but in rows, values can be of different types. I'm not sure it's possible to construct an anonymous struct using reflection, so this is as good as we can get. Users would can consume such a slice directly, doing type assertion on values, but they should expect which field has which type. Or they can implement the scanner interface in a struct, which would populate the struct. This would be more idiomatic and this is what I documented in the README.